app/javascript/components/pattern_library/wrappers/MultiSelectWrapper.js

```diff
@@ -24,7 +24,7 @@ const SelectItemShape = PropTypes.shape({
 MultiSelectWrapper.propTypes = {
   items: PropTypes.arrayOf(SelectItemShape),
   disabled: PropTypes.bool,
-  height: PropTypes.oneOf('normal', 'small')
+  height: PropTypes.oneOf(['normal', 'small'])
```

Comment:
not really in the scope of this PR, but I saw a warning in the console so I fixed this

Reply:
good catch, thanks